horizons/gui/widgets/minimap.py

```diff
@@ -51,14 +42,19 @@ def iter_minimap_points(location, world, island_color, water_color, area=None):
 
 	If `area` is set, it's supposed to be a part of `location`, that is to be
 	returned.
+
+	This function is not used anymore in the in-game minimap, but it is necessary
```

Comment:
Wording - was this function used by the in-game minimap before? If it's used only for the preview, perhaps the code could be moved there?

Reply:
I wanted to remove this function, and use the _MinimapTransform class for that but that would require some more changes to _MinimapTransform.iter_points.
It's probably still better to fix that first though.

Reply:
Then I think it's good as it is right now. I think smaller incremental changes are the way to go :).